app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    instructions = command.split()
+    source = instructions[1]
+    destination = instructions[2]
+
+    if len(instructions) != 3 or instructions[0] != "mv":
+        raise ValueError("Incorrect command")
+
+    if os.path.exists(destination):
+        raise ValueError("Already exists!")
+
+    path = "/".join(destination.split("/")[:-1])
+    new_name = "".join(destination.split("/")[-1])
```

Comment:
The use of `join` here is unnecessary since `split()` already returns the last element as a string. Consider using `destination.split('/')[-1]` directly.